storage/range.go

```diff
@@ -430,15 +430,14 @@ func (r *Range) Desc() *proto.RangeDescriptor {
 	return (*proto.RangeDescriptor)(atomic.LoadPointer(&r.desc))
 }
 
-// SetDesc atomically sets the range's descriptor. This method calls
+// setDesc atomically sets the range's descriptor. This method calls
 // processRangeDescriptorUpdate() to make the range manager handle the
-// descriptor update. Note that processRangeDescriptorUpdate()
-// acquires the metaLock.
+// descriptor update.
 //
 // This method should be called in the context of having metaLock held,
```

Comment:
Looks like this comment can go? The TODO could be moved next to the metaLock field in Range.